.github/workflows/ci.yml

```diff
@@ -36,6 +36,11 @@ jobs:
       -
         name: unit-test
         run: go test -shuffle=on $(go list ./... | grep -v -e /e2e -e /integrationtests -e /benchmarks)
+      -
+        name: helm-unittest
+        run: |
+          helm plugin install https://github.com/helm-unittest/helm-unittest.git
+          helm unittest -f 'charts/fleet/tests/*.yaml' charts/fleet
```

Comment:
I think we don't need to use `-f 'charts/fleet/tests/*.yaml'` if the test files have the `_test.yaml` suffix.